pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -208,7 +208,8 @@ func filterOutSameType(newNodeClaim *scheduling.NodeClaim, consolidate []*Candid
 		}
 	}
 
-	return filterByPrice(newNodeClaim.InstanceTypeOptions, newNodeClaim.Requirements, maxPrice)
+	filterByPrice, _ := filterByPriceWithMinValues(newNodeClaim.InstanceTypeOptions, newNodeClaim.Requirements, maxPrice)
```

Comment:
If we're ignoring a value here, I'm imagining that we need to comment why we're ignoring it. Even more than that though, I'm thinking that if we don't respect the minValues here, we could wind out in another situation where we don't respect the minValues that have been passed through. Particularly, because the `filterOutSameType` function has the potential to further filter down more instance types

Reply:
>we could wind out in another situation where we don't respect the minValues that have been passed through.

This case is already handled in the `computeConsolidation` where we try to compute `incompatibleRequirementKey` if minValues is not met before we get to `filterOutSameType`. And if the minValues is not met after `filterOutSameType`, we return empty InstanceTypeOptions and the check [here](https://github.com/kubernetes-sigs/karpenter/blob/main/pkg/controllers/disruption/multinodeconsolidation.go#L151) is going to prevent from consolidation.

I will add these details in the code.